experimental/generation/generator/packages/library/src/mergeAssets.ts

```diff
@@ -223,8 +223,10 @@ async function mergeOtherFiles(oldPath: string, oldFileList: string[], newPath:
 async function mergeRootFile(schemaName: string, oldPath: string, oldFileList: string[], newPath: string, newFileList: string[], mergedPath: string, locale: string, extension: string, oldPropertySet: Set<string>, newPropertySet: Set<string>, feedback: Feedback): Promise<void> {
     const outDir = assetDirectory(extension)
     let oldText = await fs.readFile(ppath.join(oldPath, outDir, locale, `${schemaName}.${locale}${extension}`), 'utf8')
+    oldText = oldText.replace(/\r\n\n/g, os.EOL)
```

Comment:
>\r\n\n [](start = 31, length = 6)

I'm not sure if this is the right fix.  How are we ending up with \r\n\n?  For systems that use \r that should never happen.  I suspect the issue is where that is being generated.